contrib/MockBitcoind.md

```diff
@@ -1,37 +1,56 @@
 # Run bitcoind regtest node
 
-## Create bitcoind node
+## Requirements
 
-Create a new bitcoind node with snapshot data
+Make sure you have docker and docker-compose-plugin installed and docker service is running.
 
-```bash
-make bitcoind-run
+## Initialization
+
+```sh
+make bitcoind-init
 ```
 
-## Stop bitcoind node
+Start a new terminal where you will run a container with bitcoind, by default the nativewallet will be loaded:
 
-```bash
-make bitcoind-stop
+```sh
+cd contrib; docker compose up
```

Comment:
```suggestion
cd contrib; docker compose up bitcoind
```

Reply:
this is not needed, because by default all services in the default profile will start. `btcd` is not in the default profile , so it won't start.

Reply:
Thank you for your explanation!